Assets/__Scripts/Platforms/PlatformDescriptor.cs

```diff
@@ -260,6 +260,21 @@ void HandleLights(LightsManager group, int value, MapEvent e)
         if (SoloAnEventType && e._type != SoloEventType) mainColor = invertedColor = Color.black.WithAlpha(0);
 
         IEnumerable<LightingEvent> allLights = group.ControllingLights;
+
+        if ((e._customData?.HasKey("_lightID") ?? false) && Settings.Instance.EmulateChromaAdvanced)
+        {
+            int lightID = group.EditorToGameLightIDMap.IndexOf(e._customData["_lightID"].AsInt);
+            if (lightID >= 0 && lightID < group.ControllingLights.Count)
+            {
+                allLights = new List<LightingEvent> {group.ControllingLights[lightID]};
+            }
+            else
+            {
+                Debug.LogWarning($"Light ID {lightID} does not exist for event type {e._type}!");
+                allLights = new List<LightingEvent>();
```

Comment:
I do realize that this exact same thing is being done a little bit down the page, but we can make a smaller allocation by using `Enumerable.Empty<LightingEvent>()`